docs/docs/examples/introduction.mdx

```diff
@@ -6,128 +6,42 @@ import Tabs from "@theme/Tabs";
-    label="Python"
-  />
-  <Card
-    githubLink="https://github.com/zitadel/zitadel-angular"
-    title="Angular"
-    label="Web · Mobile Web"
-  />
-  <Card
-    githubLink="https://github.com/zitadel/zitadel-nextjs"
-    title="NextJS"
-    label="Web · Mobile Web"
-  />
-  <Card
-    githubLink="https://github.com/zitadel/zitadel_flutter"
-    title="Flutter"
-    label="Android · iOS · Web · Mobile Web"
-  />
-</CardWrapper>
+- [Frontend](frontend)
+- [Backend](backend)
 
-## Libraries
+
+### Community Managed
+
+We're thrilled to have such an awesome community around ZITADEL!
+Our users are constantly pushing the boundaries and coming up with new and innovative ways to use our platform.
+
+While we haven't defined a specific scope for these projects,
+we think they're worth checking out if you're interested in exploring ZITADEL in different programming languages or frameworks.
+
+We're so grateful for our amazing community and all the awesome things they've created.
+Thanks for being a part of the ZITADEL family!
 
 | Language | Description                                                                                       | Link                                                                               |
-| -------- | ------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------- |
-| Go       | Go client library for ZITADEL.                                                                    | [https://github.com/zitadel/zitadel-go](https://github.com/zitadel/zitadel-go)     |
-| .Net     | Authentication / Authorization library written in dotnet for the asp.net web application package. | [https://github.com/zitadel/zitadel-net](https://github.com/zitadel/zitadel-net)   |
-| Dart     | Dart library for ZITADEL, contains gRPC and API access elements.                                  | [https://github.com/zitadel/zitadel-dart](https://github.com/zitadel/zitadel-dart) |
+| -------- | ------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------- |
+| Dart     | Dart library for ZITADEL, contains gRPC and API access elements.                                  | [https://github.com/smartive/zitadel-dart](https://github.com/smartive/zitadel-dart) |
 | Elixir   | API Client for the ZITADEL API.                                                                   | [https://github.com/jshmrtn/zitadel_api](https://github.com/jshmrtn/zitadel_api)   |
+| NodeJS   | Authentication Helper for NodeJS                                                                 | [https://github.com/buehler/node-passport-zitadel](https://github.com/buehler/node-passport-zitadel)   |
+| NextAuth.js | NextAuth Provider for ZITADEL | [https://next-auth.js.org/providers/zitadel](https://next-auth.js.org/providers/zitadel) |
```

Comment:
I would not mix these with SDKs but have them under Middleware / More, as explained here in the issue.